src/inferences/junctionTree/create-initial-potentials.ts

```diff
@@ -63,13 +85,14 @@ const getPotentialValueForNodeWithoutParents = (combination: ICombinations, node
   return cpt[combinationValue]
 }
 
-const getPotentialValueForNode = curry((combination: ICombinations, node: INode) => {
-  if (hasNodeParents(node)) {
-    return getPotentialValueForNodeWithParents(combination, node)
-  }
+const getPotentialValueForNode: (combination: ICombinations) => (node: INode) => number =
```

Comment:
added an explicit type signature.   Some of the tests were failing to compile because they could not infer the correct type.